roles/openshift_gcp/files/openshift-bootstrap-update.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
```

Comment:
Move to openshift_bootstrap_autoapprover role. This is useful outside of gcp!

Reply:
This part depends on GCP (because of project metadata), but will move the other parts.